### DISCUSSION TITLE
From notice type 4 to 36,26,20 and 25

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#579] [Q&A] [answered]
Hi Lab
I'am making same test in preview env,
can you tell me please the reason why from notice type 4 ( with continue procedure ) I can make a notice type 36, 26. 20 and 25 ?

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/dac8265a-196a-489d-a8b2-aa2af527eaf4)

I guess that 4, 16, 20, 25 and 36 are all notice General Procurement but notice 33, for example, is G.P. as well

we would like to understand the possible links between eforms2 notice
Thanks a lot
Renato

**Comment by rengit62** [comment]
From notice 11 => notice 30 both Sectoral Procurement, but why only 30 ?

**Comment by YvesJo** [comment, marked as answer]
Hi Renato,
For questions regarding eNotices2, please contact the Helpdesk at [ted@publications.europa.eu](mailto:ted@publications.europa.eu).
